### DISCUSSION TITLE
Could you make it run with GPU

### DISCUSSION THREAD

**Discussion by vuongmanh15** [SWHL/RapidVideOCR#42] [Ideas]
It will run faster with longer video

**Comment by SWHL** [comment]
I am currently exploring possible solutions to use GPU inference accelearation, please continue to pay attention.

**Reply by tvone** [reply]
> I am currently exploring possible solutions to use GPU inference accelearation, please continue to pay attention.

How's it going?

**Reply by SWHL** [reply]
In version v3.0.1, `rapid_videocr` now supports GPU inference.
The specific usage method is the same as `rapidocr>=2.0.0`. GPU inference can be performed by passing ocr_params. Please refer to the official documentation of RapidOCR for details. [docs](https://rapidai.github.io/RapidOCRDocs/main/install_usage/rapidocr/usage/#_4)
